rust/extractor/src/rust_analyzer.rs

```diff
@@ -95,7 +95,7 @@ impl<'a> RustAnalyzer<'a> {
 
                 ParseResult {
                     ast: source_file,
-                    text: input.text(semantics.db),
+                    text: input.text(semantics.db).clone(),
```

Comment:
Why do we need to `clone` now?

Reply:
they changed this to return a ref to an `Arc`, instead of directly a moved `Arc` like it was before, so we now need to explicitly clone it. It remains an `Arc`, so cloning is cheap.